src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -330,8 +360,140 @@
+              <input id="frontend_port"
+                     class="form-control"
+                     type="number"
+                     formControlName="frontend_port"
+                     min="1"
+                     max="65535">
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('frontend_port', frm, 'pattern')"
+                    i18n>The entered value needs to be a number.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('frontend_port', frm, 'min')"
+                    i18n>The value must be at least 1.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('frontend_port', frm, 'max')"
+                    i18n>The value cannot exceed 65535.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('frontend_port', frm, 'required')"
+                    i18n>This field is required.</span>
+            </div>
+          </div>
+
+          <!-- monitor_port -->
+          <div class="form-group row">
+            <label class="cd-col-form-label"
+                   [ngClass]="{'required': ['ingress'].includes(serviceForm.controls.service_type.value)}"
+                   for="monitor_port">
+              <span i18n>Monitor Port</span>
+              <cd-helper>
+                <span i18n>The port used by haproxy for load balancer status.</span>
+              </cd-helper>
+            </label>
+            <div class="cd-col-form-input">
+              <input id="monitor_port"
+                     class="form-control"
+                     type="number"
+                     formControlName="monitor_port"
+                     min="1"
+                     max="65535">
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('monitor_port', frm, 'pattern')"
+                    i18n>The entered value needs to be a number.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('monitor_port', frm, 'min')"
+                    i18n>The value must be at least 1.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('monitor_port', frm, 'max')"
+                    i18n>The value cannot exceed 65535.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('monitor_port', frm, 'required')"
+                    i18n>This field is required.</span>
+            </div>
+          </div>
+          <!-- virtual_interface_networks -->
+          <div class="form-group row">
+            <label class="cd-col-form-label"
+                   for="virtual_interface_networks">
+              <span i18n>CIDR Networks</span>
+              <cd-helper>
+                <span i18n>A list of networks to identify which ethernet interface to use for the virtual IP.</span>
```

Comment:
```suggestion
                <span i18n>A list of networks to identify which network interface to use for the virtual IP address.</span>
```

Reply:
Ok, will add this suggestion. I just followed what's mentioned in docs 
https://docs.ceph.com/en/latest/cephadm/rgw/#high-availability-service-for-rgw